improver/cli/gradient_between_adjacent_grid_squares.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# (C) Crown copyright, Met Office. All rights reserved.
+#
+# This file is part of IMPROVER and is released under a BSD 3-Clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""CLI to calculate the gradient between adjacent grid squares."""
+from improver import cli
+
+
+@cli.clizefy
+@cli.with_output
+def process(cube: cli.inputcube, *, regrid: bool = False):
+    """Calculate the gradient between adjacent grid squares within
+    a cube. The gradient is calculated along the x and y axis
+    individually.
+
+    Args:
+        cube (iris.cube.Cube):
+            Cube from which the gradients will be calculated.
+        regrid (bool):
+                If True, the gradient cube is regridded to match the spatial
```

Comment:
The indentation here has gone strange again. Is this something black/flake is doing?

Reply:
I suspect it is just me getting it wrong